/**
 * Creates a new instance of {@code ConcurrentRuntimeException} and
 * initializes it with the given cause.
 *
 * @param cause the cause of this exception
 * @throws IllegalArgumentException if the cause is not a checked exception
 */
public ConcurrentRuntimeException(Throwable cause) {
    super(ConcurrentUtils.checkedException(cause));
}